### PR TITLE
CAMS-135 attorney information

### PR DESCRIPTION
**Pull request by fmaddenflx**
# Purpose

Add Debtor Attorney information to Case Details screen

# Major Changes

Debtor Attorney name, address, and phone number was added to Case Details screen

# Testing/Validation

Testing coverage is above 90% and both backend and front end.

# Notes

Dev-tools were updated to provide both SQL and JSON data from the same set of fixtures.  Further work is needed to complete auto-generation for the tax-id and ssn.  Code seems to be present but isn't working correctly.

### CODE REVIEW

File: backend/functions/lib/adapters/gateways/dxtr/cases.dxtr.gateway.ts [hunk 1]
```diff
@@ -26,6 +27,7 @@ const closedByCourtTxCode = 'CBC';
 const dismissedByCourtTxCode = 'CDC';
 const reopenedDate = 'OCO';
 
+//TODO: Add data call to AO_AT to get debtor attorney information
```
Comment (jamesobrooks) [inline backend/functions/lib/adapters/gateways/dxtr/cases.dxtr.gateway.ts:30]: This should be removed.

File: dev-tools/README.md [hunk 2]
```diff
@@ -6,7 +6,7 @@
 npm run test-data
```
Comment (jamesobrooks) [inline dev-tools/README.md:6]:
```suggestion
npm run test-data:[sql | json]
```

File: dev-tools/README.md [hunk 3]
```diff
@@ -6,7 +6,7 @@
...
 ```
 
-SQL statements are written to `data/test-data.sql`.
+SQL statements are written to `data/test-data.json`.
```
Comment (jamesobrooks) [inline dev-tools/README.md:9]:
```suggestion
SQL statements are written to `data/test-data.sql` when `test-data:sql` is run.
JSON objects are written to `data/test-data.json` when `test-data:json` is run.
```

File: user-interface/src/components/CaseDetail.tsx [hunk 4]
```diff
@@ -173,16 +173,72 @@ export const CaseDetail = (props: CaseDetailProps) => {
...
                     >
-                      {caseDetail.debtor.address4}
+                      {caseDetail.debtor.cityStateZipCountry}
                     </div>
                   )}
                 </div>
               </div>
+              <div className="debtor-counsel-information padding-bottom-4 case-card">
+                <h3>Debtor Counsel</h3>
+                {caseDetail.debtorAttorney && (
```
Comment (btposey) [inline user-interface/src/components/CaseDetail.tsx:188]: We should either 1. Not show the header abover OR 2. Adda note that there is not debtor attorney. Or pro se.

Comment (btposey) [reply user-interface/src/components/CaseDetail.tsx:188]: Phoenix: Show copy when the attorney is not available under the header. "Information is not available at this time."

Comment (btposey) [reply user-interface/src/components/CaseDetail.tsx:188]: We should make this copy consistent for all info blocks n the detail screen where data is not present.

**Review by btposey — APPROVED** [review]
Approved conditionally depending on the changes to the CaseDetail.ts source.

**Review by amorrow-flexion — APPROVED** [review]

**Pull request merged**